Sources/Core/API/Animation.swift

```diff
@@ -78,7 +78,7 @@ public extension Animation {
 
     /// Initialize an instance with an array of images to use for the animation
     init(images: [Image], frameDuration: TimeInterval) {
-        content = .textures(images.map(Texture.init))
+        content = .textures(images.map { Texture(image: $0) })
```

Comment:
Yeah, it was exactly because of new param was added to the init and we couldn't use that shorthand form inside `map`. But since we decided to get rid of `format` param for pre-loaded images, we can safely revert it back 🙃